hub/models.py

```diff
@@ -388,6 +388,17 @@ def shader_value(self, area):
         return None, None, None
 
 
+class AreaType(models.Model):
```

Comment:
PS @alexander-griffen I learned a few weeks ago that you can pass a string to `models.ForeignKey()` when you want to refer to a model that’s defined _further down_ in the file. So, in this case, a line like…

```python
area_type = models.ForeignKey("AreaType", on_delete=models.CASCADE, null=True)
```

…would have saved you having to shuffle this Class further up the file.